static/citus/v2-chinook-deployment.json

```diff
@@ -1683,623 +1683,4484 @@
+          "operatorName": "boolle",
+          "argumentType": "bool",
+          "isInfix": false
+        },
+        "boollt": {
+          "operatorName": "boollt",
+          "argumentType": "bool",
+          "isInfix": false
+        },
+        "boolne": {
+          "operatorName": "boolne",
+          "argumentType": "bool",
+          "isInfix": false
+        },
+        "boolor_statefunc": {
+          "operatorName": "boolor_statefunc",
+          "argumentType": "bool",
+          "isInfix": false
+        },
+        "pg_promote": {
+          "operatorName": "pg_promote",
+          "argumentType": "int2",
+          "isInfix": false
         }
       },
       "char": {
         "_eq": {
           "operatorName": "=",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_gt": {
           "operatorName": ">",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_gte": {
           "operatorName": ">=",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_ilike": {
           "operatorName": "~~*",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_iregex": {
           "operatorName": "~*",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_like": {
           "operatorName": "~~",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_lt": {
           "operatorName": "<",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_lte": {
           "operatorName": "<=",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_neq": {
           "operatorName": "<>",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_nilike": {
           "operatorName": "!~~*",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_niregex": {
           "operatorName": "!~*",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_nlike": {
           "operatorName": "!~~",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_nregex": {
           "operatorName": "!~",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
         },
         "_regex": {
           "operatorName": "~",
-          "argumentType": "char"
+          "argumentType": "char",
+          "isInfix": true
+        },
+        "alter_role_if_exists": {
```

Comment:
maybe we should filter functions like this one.

Reply:
Yeah that's true. The PR now filters certain types and functions which are to do with server administraton.